### DISCUSSION TITLE
Stuck at Loading Graphic-Walker UI...

### DISCUSSION THREAD

**Discussion by LamChiho** [Kanaries/pygwalker#557] [Q&A] [answered]
Below are codes used, and I keep on getting Loading Graphic-Walker UI... 
I don't have any idea about what is going on. 
from pygwalker.api.streamlit import StreamlitRenderer
import pandas as pd
import streamlit as st
from IPython.display import display
import ipywidgets as widgets

# Adjust the width of the Streamlit page
st.set_page_config(
    page_title="Use Pygwalker In Streamlit",
    layout="wide"
)

# Add Title
st.title("Use Pygwalker In Streamlit")

# You should cache your pygwalker renderer, if you don't want your memory to explode
@st.cache_resource
def get_pyg_renderer() -> "StreamlitRenderer":
    df = pd.read_csv("./iris.csv")
    # If you want to use feature of saving chart config, set `spec_io_mode="rw"`
    return StreamlitRenderer(df, spec="./gw_config.json", spec_io_mode="rw")

renderer = get_pyg_renderer()

renderer.explorer()

#streamlit run pygtester.py

**Comment by longxiaofei** [comment, marked as answer]
Hi @LamChiho 

Could you provide the browser log after opening the streamlit page?

![image](https://github.com/Kanaries/pygwalker/assets/28337703/a5635dda-73b9-4438-84cd-5c247380c7fa)

**Reply by LamChiho** [reply]
![8cb2bcf3254d853d949c6ce1fc05207d](https://github.com/Kanaries/pygwalker/assets/37561723/84b6ef50-cdf4-455b-ac9c-8d9c642c591f)
Sorry forgot to paste that. Thank you for your response.

**Reply by longxiaofei** [reply]
I need your browser logs to help me troubleshoot.

Like the red box position in my picture(Each browser may have a different way of opening developer tools.).

**Reply by LamChiho** [reply]
I tried to open the browser log on my default browser and failed, and I switched browser to chrome and it was fixed. Sorry for bothering you. And big thanks for your help.

**Reply by longxiaofei** [reply]
🎉🎉🎉

I want to reproduce this bug and find out the specific cause.

Could you provide me with what your default browser is?

Thanks.

**Reply by LamChiho** [reply]
Sure, I was using QQ browser.

**Reply by uhleg** [reply]
> ![8cb2bcf3254d853d949c6ce1fc05207d](https://private-user-images.githubusercontent.com/37561723/333134147-84b6ef50-cdf4-455b-ac9c-8d9c642c591f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMyMDMwNDIsIm5iZiI6MTcyMzIwMjc0MiwicGF0aCI6Ii8zNzU2MTcyMy8zMzMxMzQxNDctODRiNmVmNTAtY2RmNC00NTViLWFjOWMtOGQ5YzY0MmM1OTFmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODA5VDExMjU0MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRhMzY2MWE3OTkxZWY3OWY3YWUyMWNkZDc5ZmFmMDc1ZTcyY2MxMDhhMWMzM2I5YTAzYzAwMTRiMDk5Njk1ZGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.5k6B6jWcnUnNMv-Q5mPtohIyvPOuHMELXgWmCm2mQe0) Sorry forgot to paste that. Thank you for your response.对不起，忘了粘贴那个。感谢您的回复。

可以说一下忘粘贴什么了吗？我遇到了跟您同样的问题
感谢
